lib/phoenix_live_view/helpers.ex

```diff
@@ -379,8 +379,18 @@ defmodule Phoenix.LiveView.Helpers do
 
   The following errors may be returned:
 
-    * `:too_large` - The entry exceeds the `:max_file_size` constraint
     * `:too_many_files` - The number of selected files exceeds the `:max_entries` constraint
+  """
+  def upload_errors(%Phoenix.LiveView.UploadConfig{} = conf) do
```

Comment:
We could add some docs here too, wdyt?

```suggestion

  ## Examples

      def error_to_string(:too_many_files), do: "You have selected too many files"

      <%= for err <- upload_errors(@uploads.avatar) do %>
        <div class="alert alert-danger">
          <%= error_to_string(err) %>
        </div>
      <% end %>
  """
  def upload_errors(%Phoenix.LiveView.UploadConfig{} = conf) do
```